test/Microsoft.Azure.WebJobs.Host.UnitTests/Scale/ProcessMonitorTests.cs

```diff
@@ -51,6 +52,62 @@ public ProcessMonitorTests()
                 }
             };
 
+        [Fact]
+        public void Process_TotalProcessorTime_ReturnsExpectedResult()
+        {
+            var process = Process.GetCurrentProcess();
+            var monitor = new ProcessMonitor(process);
+
+            Assert.Equal(monitor.Process.TotalProcessorTime, process.TotalProcessorTime);
+            Assert.Equal(process.TotalProcessorTime, process.TotalProcessorTime);
+
+            process.Refresh();
+
+            Assert.Equal(process.TotalProcessorTime, process.TotalProcessorTime);
+        }
+
+        [Fact]
+        public void Process_PrivateMemoryBytes_ReturnsExpectedResult()
+        {
+            var process = Process.GetCurrentProcess();
+            var monitor = new ProcessMonitor(process);
+
+            Assert.Equal(monitor.Process.PrivateMemoryBytes, process.PrivateMemorySize64);
+            Assert.Equal(monitor.Process.PrivateMemoryBytes, process.PrivateMemorySize64);
+
+            process.Refresh();
+
+            Assert.Equal(monitor.Process.PrivateMemoryBytes, process.PrivateMemorySize64);
+        }
+
+        [Fact]
+        public void Process_Id_ReturnsExpectedResult()
+        {
+            var process = Process.GetCurrentProcess();
+            var monitor = new ProcessMonitor(process);
+
+            Assert.Equal(monitor.Process.Id, process.Id);
```

Comment:
Is this just testing the property accessor? It feels a little unnecessary.

Reply:
Yeah, it's testing the wrapper abstraction ensuring that all the wrappers return the expected values. I included this test for completness.